src/aioquic/h3/connection.py

```diff
@@ -194,19 +200,46 @@ def stream_is_request_response(stream_id: int):
     return stream_id % 4 == 0
 
 
+def validate_key(key: bytes):
```

Comment:
validate_header_name might be a better name?

Also maybe explicitly type the return value to None and a docstring?

Reply:
can do